src/main/java/org/ow2/proactive/catalog/repository/CatalogObjectRevisionRepository.java

```diff
@@ -43,14 +45,17 @@
 public interface CatalogObjectRevisionRepository extends JpaRepository<CatalogObjectRevisionEntity, UUID>,
         JpaSpecificationExecutor<CatalogObjectRevisionEntity> {
 
-    @Query("SELECT cor FROM CatalogObjectRevisionEntity cor WHERE cor.catalogObject.bucket.bucketName in ?1 AND cor.catalogObject.lastCommitTime = cor.commitTime")
-    List<CatalogObjectRevisionEntity> findDefaultCatalogObjectsInBucket(List<String> bucketNames);
+    @Query("SELECT cor FROM CatalogObjectRevisionEntity cor WHERE cor.projectName = null OR cor.projectName = ''")
+    List<CatalogObjectRevisionEntity> findByProjectNameIsNullOrIsEmpty();
```

Comment:
I think it would be clearer if you rename if `findEmptyOrNullProjectName` or `findWithEmptyOrNullProjectName`